examples/python/for_satellite_operators/tlm_and_cmd_stream_extended.py

```diff
@@ -0,0 +1,161 @@
+# Copyright 2023 Infostellar, Inc.
+# Opens a stream to both receive telemetry and send commands.
```

Comment:
I agree it is not clear what the driver behind creating this extended example is. If there's only a few changes (I see at least the multiple attempts change) then I think it's fine to simply update tlm_and_cmd_stream.py instead of break this out into it's own example